src/core/structs.rs

```diff
@@ -460,6 +460,112 @@ impl<const UID_LENGTH: usize> Serializable for EncryptedTable<UID_LENGTH> {
     }
 }
 
+#[derive(Default)]
+pub struct EncryptedMultiTable<const UID_LENGTH: usize>(HashMap<Uid<UID_LENGTH>, Vec<Vec<u8>>>);
+
+impl<const UID_LENGTH: usize> EncryptedMultiTable<UID_LENGTH> {
+    pub fn with_capacity(capacity: usize) -> Self {
+        Self(HashMap::with_capacity(capacity))
+    }
+
+    pub fn to_encrypted_table(
+        self,
+        debug: &'static str,
+    ) -> Result<EncryptedTable<UID_LENGTH>, FindexErr> {
+        let mut table = EncryptedTable::with_capacity(self.0.len());
```

Comment:
Shouldn't this work?
```suggestion
        let mut table = EncryptedTable::with_capacity(self.len());
```

Reply:
Fix in 30943dc